src/components/toolbar/OrganizationToolBar.tsx

```diff
@@ -0,0 +1,79 @@
+import {Toolbar, ToolbarContent, ToolbarItem} from '@patternfly/react-core';
+import {DropdownCheckbox} from './DropdownCheckbox';
+import {DropdownFilter} from './DropdownFilter';
+import {FilterBar} from './FilterBar';
+import {ToolbarButton} from './ToolbarButton';
+import {Kebab} from './Kebab';
+import {ToolbarPagination} from './Pagination';
+import {filterOrgState} from 'src/atoms/UserState';
+import {useRecoilState} from 'recoil';
+
+export function OrganizationToolBar(props: OrganizationToolBarProps) {
```

Comment:
What do you think about moving `OrganizationToolBar.tsx` and `RepositoryToolBar.tsx` out of the components folder? They are specific implementations of a toolbar, not really reusable like some of the other components.

Reply:
Yes, makes sense. I will do that.